src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/get_system_capability_request.cc

```diff
@@ -163,6 +163,17 @@ void GetSystemCapabilityRequest::Run() {
                              all_services);
       break;
     }
+    case mobile_apis::SystemCapabilityType::DRIVER_DISTRACTION:
+      if (hmi_capabilities.driver_distraction_capability() &&
+          hmi_capabilities.driver_distraction_supported()) {
```

Comment:
Seems checking `driver_distraction_supported` might be redundant here. This value should be used in the RAI response (see https://github.com/smartdevicelink/rpc_spec/pull/263/files#r453856989)